python/ql/src/semmle/python/frameworks/Django.qll

```diff
@@ -2029,7 +2029,7 @@ private module Django {
     DjangoViewClassDef() { this.getABase() = django::views::generic::View::subclassRef().asExpr() }
 
     /** Gets a function that could handle incoming requests, if any. */
-    DjangoRouteHandler getARequestHandler() {
+    Function getARequestHandler() {
```

Comment:
Is this change related/necessary?

Reply:
This change was necessary to get `DjangoRouteHandler` into a good state. Without this change, I saw that _no_ request handlers on classes was included in the `DjangoRouteHandler` class. I assume this is because the `getARequestHandler` predicate now will only give you results that are already considered a `DjangoRouteHandler`, so essentially we can remove the second part of the char-pred (since it doesn't specify any new functions that should be included in the `DjangoRouteHandler` class)

```diff
DjangoRouteHandler() {
 exists(DjangoRouteSetup route | route.getViewArg() = djangoRouteHandlerFunctionTracker(this))
-or
-any(DjangoViewClass vc).getARequestHandler() = this
}
```

Reply:
Ah, so `getARequestHandler` is the more basic predicate.